### DISCUSSION TITLE
how not to display vertex stroke?

### DISCUSSION THREAD

**Discussion by reuse-ay** [maxGraph/maxGraph#406] [Q&A] [answered]
According to [maxGraph document](https://maxgraph.github.io/maxGraph/api-docs/types/CellStateStyle.html), you can use no color for stroke by setting `strokeColor` to `"none"`. I added `strokeColor:"none"` to `style`  in `insertVertex`. But stroke is still displayed. I want hide stroke.

reproduce:[here](https://stackblitz.com/~/github.com/reuse-ay/angular_practice?file=src/app/content/graph-menu/graph-menu.component.ts)
Please select Maxgraph Menu from upper left menu in preview.
![image](https://github.com/maxGraph/maxGraph/assets/150884298/9366de13-adee-4974-ba10-fd0bbbd49673)

**Comment by reuse-ay** [comment]
`fillColor:"none"` also doesn't work. Is this caused by "none" issue?

**Reply by tbouffard** [reply]
Hi
This behavior has been introduced in https://github.com/maxGraph/maxGraph/pull/383 (v0.10.1) by myself 😸 and this seems to be an issue.
The maxGraph behavior was to not have color when set to none: https://github.com/jgraph/mxgraph/blob/v4.2.2/javascript/src/js/view/mxStylesheet.js#L42-L43
#383 set the value to the one in the default or base name styles. I will provide a fix soon.

In the meantime, the workarounds proposed in this discussion can be used (strokeOpacity set to zero or strokeColor set to transparent).

**Reply by tbouffard** [reply]
Proposed fix:  #409

**Comment by reuse-ay** [comment]
If vertex has `fillColor` other than `"none"`, vertex seems to have no stroke by setting `strokeColor` to same color to `fillColor`. Default vertex `fillColor` is c3d9ff.

**Comment by reuse-ay** [comment]
You can use "strokeOpacity". Be careful that range is 0-100.

**Comment by reuse-ay** [comment, marked as answer]
`strokeColor:"transparent"` seems to work.

**Comment by tbouffard** [comment]
The issue is fixed in https://github.com/maxGraph/maxGraph/releases/tag/v0.10.2 thanks to #409.
